mkt/developers/utils.py

```diff
@@ -154,3 +154,14 @@ def escalate_prerelease_permissions(app, validation, version):
         escalate_app(
             app, version, nobody, 'App uses prerelease permissions',
             mkt.LOG.ESCALATION_PRERELEASE_APP)
+
+
+def prioritize_app(app, user):
+    if not app.priority_review:
```

Comment:
I'm tempted to say take the update out of here and rename this `notify_app_prioritization` or something. I do like having a function you can call and have it do everything you need to prioritize an app though. You could also remove `priority_review` from the `updates` dict above and just live with the fact that we're going to do two updates. I'm not sure if there are any real downsides to doing two updates.